Formula/x/xcode-build-server.rb

```diff
@@ -0,0 +1,30 @@
+class XcodeBuildServer < Formula
+  desc "Build server protocol implementation for integrate xcode with sourcekit-lsp"
+  homepage "https://github.com/SolaWing/xcode-build-server"
+  url "https://github.com/SolaWing/xcode-build-server/archive/refs/tags/v1.0.0.tar.gz"
+  sha256 "90ea9b8a3e89dfa3a0bf045236c90512f5faac77b76f8039b7937bc6b34d25e0"
+  license "MIT"
+  head "https://github.com/SolaWing/xcode-build-server.git", branch: "master"
+
+  # For reading Xcode logs archives we need to unzip them
+  depends_on "gzip"
+
+  # It's a macOS only tool because it's not designed to work without Xcode
+  depends_on :macos
+
+  # Script is written in Python
+  depends_on "python@3.9"
```

Comment:
```suggestion
  depends_on :macos
  depends_on "python@3.12"
```

Reply:
We use the latest version of Python that we can. If this cannot use higher than Python 3.9, we won't be able to add it to Homebrew at this point.

Reply:
hank you, I've updated to the latest version of Python. I just didn't understand correctly that I should use the latest version instead of the lowest possible one.